cookiecutter.json

```diff
@@ -1,12 +1,12 @@
 {
-    "base_app_name": "CMSApp"
-    , "repo_name": "repo_name"
-    , "project_name": "Project X"
-    , "github_username": "Fueled"
-    , "github_reponame": ""
-    , "project_description": "add a short project description here"
-    , "timezone": "UTC"
-    , "version": "0.1.0"
-    , "api_base_url": "http://localhost:8000/api/v1/"
-    , "api_auth_url": "auth"
+    "base_app_name": "Hello World",
+    "repo_name": "Hello World",
+    "project_name": "Hello World",
```

Comment:
We can use something like this https://github.com/Fueled/django-init/blob/master/cookiecutter.json#L4 here.